tests/fixtures/cassettes/rcodezero/IntegrationTests/test_provider_authenticate_with_unmanaged_domain_should_fail.yaml

```diff
@@ -0,0 +1,55 @@
+interactions:
+- request:
+    body: !!python/unicode '{}'
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '2'
+      Content-Type:
+      - application/json
+      User-Agent:
+      - python-requests/2.22.0
+    method: GET
+    uri: https://my.rcodezero.at/api/v1/zones/thisisadomainidonotown.com
+  response:
+    body:
+      string: !!python/unicode '{"status":"failed","message":"Zone not found"}'
+    headers:
+      cache-control:
+      - no-cache, private
+      connection:
+      - Keep-Alive
+      content-length:
+      - '46'
+      content-type:
+      - application/json
+      date:
+      - Mon, 30 Sep 2019 13:25:40 GMT
+      keep-alive:
+      - timeout=5, max=100
+      server:
+      - Apache
+      set-cookie:
```

Comment:
Maybe this is something you want to filter out from the records?

Reply:
Hm, I'm not getting the point. What is the problem with the response data?

Reply:
There is no problem, in fact. It is just that since this is a CSRF protection cookie, maybe it contains some sensible data, and maybe you would like to configure the cassettes filtering to remove this value from the yml files committed in the repo.

Reply:
The cookies should not contain sensible data, but I excluded them from the cassesstes files because they are not needed for the testing.